Sources/FeaturevisorSDK/Instance.swift

```diff
@@ -92,23 +92,23 @@ let emptyDatafile = DatafileContent(
 
 public class FeaturevisorInstance {
     // from options
-    private var bucketKeySeparator: String
-    private var configureBucketKey: ConfigureBucketKey?
-    private var configureBucketValue: ConfigureBucketValue?
-    private var datafileUrl: String?
-    private var handleDatafileFetch: DatafileFetchHandler?
-    private var initialFeatures: InitialFeatures?
-    private var interceptContext: InterceptContext?
-    private var logger: Logger
-    private var refreshInterval: TimeInterval?  // seconds
-    private var stickyFeatures: StickyFeatures?
+    internal var datafileUrl: String?
```

Comment:
`internal` is the default, I don't think you need to specify it

Reply:
Yes, it's the default one. It was my personal feeling to point out under this SDK that these variables have an internal type. I will think and maybe will go without this "internal" keywoard but it would be under different PR